regression/contracts/assigns_enforce_structs_07/test.desc

```diff
@@ -3,13 +3,18 @@ main.c
 --enforce-contract f1 --enforce-contract f2 _ --malloc-may-fail --malloc-fail-null --pointer-check
 ^EXIT=10$
 ^SIGNAL=0$
-^\[f1.\d+\] line \d+ Check that p->buf\[\(.*\)0\] is assignable: SUCCESS$
+^\[f1.\d+\] line \d+ Check that p->buf\[\(.*\)0\] is assignable: (SUCCESS|FAILURE)$
```

Comment:
So we don't care what the verification result is? Would you actually want to see both of them? If so, two lines will be required.

Reply:
We want to see the target validity at line 8 fail, which was missing, thanks!